docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,178 @@
 ---
-title: Smart Optimistic Rollups
-authors: 'Nomadic Labs, TriliTech'
+title: Smart Rollups
+authors: 'Nomadic Labs, TriliTech, Tim McMackin'
```

Comment:
I'm not sure we list individual contributors as authors in documentation.

Reply:
We're not consistent about this; some files list organizations, other list individual authors.

Reply:
I'll straighten this out in the future, but for now it's not shown anywhere on the site and we can ignore it.